RELEASE.md

```diff
@@ -17,6 +17,22 @@ Please follow the established format:
 
 <!-- Add release notes for the upcoming release here -->
 
+# Release 3.11.0
+
+## Major features and improvements
+- Allow the selection and filtering of nodes by modular pipeline on the flowchart via the sidebar under the categories section. This includes changes to both the server to include modualr pipeline data in the responses, as well as front end changes to enable the new modular pipeline data type. (#391, #394, #401, #402, #408, #410, #421)
+- Addition of the Architecture docs. (#382, #393)
+- Add metadata to random data generator. (#397)
+- Simplify layout algorithm, improve layout quality and performance. (#398)
+- Improve layer solving approach when layers partially defined. (#407)
+- Removal of 'code' flag to enable the code panel feature by default. (#404)
+- Removal of 'lazy' flag to enable lazy loading of the sidebar by default. (#404)
```

Comment:
Nit: Please use the imperative tense! e.g.
- "Addition of the Architecture docs." -> "Add Architecture docs."
- "Removal of 'code' flag to enable the code panel feature by default." -> "Remove 'code' flag to enable the code panel feature by default."